### DISCUSSION TITLE
请问下有没有类似 https://www.easywechat.com/ 网站这种扫描二维码关注公众号登陆的教程

### DISCUSSION THREAD

**Discussion by gdgeek** [w7corp/easywechat#2278] [Q&A]
请问下有没有类似 https://www.easywechat.com/ 网站这种扫描二维码关注公众号登陆的教程

**Comment by overtrue** [comment]
> 请问下有没有类似 https://www.easywechat.com/ 网站这种扫描二维码关注公众号登陆的教程

这个网站连登录都没有，哪里来的扫码登录啊

**Reply by Alex37882388** [reply]
大佬，你这网站为啥老是打不开呢？ 用V皮恩也是打不开呢，这点小事儿，微擎得赞助一下啊，不然想赞助的人多了去的吧。

**Reply by overtrue** [reply]
备案中 https://easywechat.vercel.app/
